addons/payment_adyen/controllers/main.py

```diff
@@ -271,6 +271,8 @@ def adyen_notification(self):
                 notification_data['resultCode'] = 'Authorised'
             elif event_code == 'CANCELLATION':
                 notification_data['resultCode'] = 'Cancelled'
+            elif event_code == 'REFUND':
+                notification_data['resultCode'] = 'Refund'
```

Comment:
Same as above. You need to find a clean and simple way to process feedback data from a refund notification.